docs/content/Getting Started Using.fsx

```diff
@@ -0,0 +1,179 @@
+(**
+---
+category: end-users
+categoryindex: 1
+index: 1
+---
+
+# Getting started using analyzers
+
+## Premise
+
+We assume the analyzers you want to use are distributed as a nuget package.
+
+## Using analyzers in a single project
+
+First, we need to add the [fsharp-analyzers](https://www.nuget.org/packages/fsharp-analyzers) dotnet tool to the tool-manifest.
+```shell
+dotnet tool install fsharp-analyzers
+```
+
+Next, add the `PackageReference` pointing to your favorite analyzers to the `.fsproj` file of the project you want to analyze:
+
+```xml
+<PackageReference Include="G-Research.FSharp.Analyzers" Version="0.1.6">
+    <PrivateAssets>all</PrivateAssets>
+    <IncludeAssets>build</IncludeAssets>
+</PackageReference>
+```
+
+At the time of writing, the [G-Research analyzers](https://github.com/g-research/fsharp-analyzers) [package](https://www.nuget.org/packages/G-Research.FSharp.Analyzers) contains the only analyzers compatible with the latest CLI tool.  
+Finally, add a custom MSBuild target to the `.fsproj` file for easy invocation of the analyzer:
```

Comment:
I think this needs a little bit more build-up.
We use the MSBuild target because the `PkgG-Research_FSharp_Analyzers` is convenient.
Maybe we should first give an example of a command line invocation with a hardcoded path.
And thus point out that it is a bit tricky to find that absolute path on each developer's machine.